README.md

```diff
@@ -729,6 +732,7 @@ see also [Releases](https://github.com/Dynatrace/OneAgent-SDK-for-dotnet/release
 
 |Version    |Description                                  |
 |:----------|:--------------------------------------------|
+|1.6.0      |Adds W3C Trace Context support (`IOutgoingWebRequestTracer.InjectTracingHeaders`)|
```

Comment:
Maybe link to https://www.w3.org/TR/trace-context/ from here.

Reply:
good idea, I'll add the link to the release notes